helm/cluster-autoscaler-app/templates/deployment.yaml

```diff
@@ -23,12 +23,14 @@ spec:
       - effect: NoSchedule
         operator: "Exists"
         key: node-role.kubernetes.io/master
+      {{- if ((.Values.node).nodeSelector) }}
```

Comment:
Still no parentheses needed here as long as nobody actively sets `.Values.node` to `nil`. Other than that the defaults in `values.yaml` are getting used and prevent `.Values.node` from being `nil`.

Reply:
I don't mind having a safeguard tbh :)